### ISSUE TITLE
Add support for DeathLog

### ISSUE THREAD

**Issue by AlexBrightwater** [TheCSDev/mc-better-stats#13]
It would be super cool if there was compatibility for [DeathLog](https://modrinth.com/mod/deathlog).

Currently, when opening the Stats screen, the DeathLog button is hidden till one clicks on the button to show the vanilla stats. So an implementation of the DeathLog Button in the BSS GUI would be awesome.

**Comment by TheCSDev** [comment]
Hello, and thank you for the suggestion.
An interesting mod, even tho I've never heard of it before. I think I will place a button that leads to that mod, altho I am unsure of where to make room for that button, as well as any buttons for any other mods in the future. The left side panel technically doesn't have any space left, as it gets completely filled when resizing the Minecraft window to a smaller size, and I feel like creating another tab just for those special buttons would make the mod have too many tabs to cycle thru. I guess I will think of something once I get the time to.
In the meantime, have a great day and/or night.

**Comment by TheCSDev** [comment]
I attempted to implement this, however my attempt appears to have failed. The mod also does not provide any API or any way to open their Death Log screen from another mod. Was planning this for v2.0, but will probaby delay it until later, for when I actually figure out how to get it working.

**Comment by QueenUlyana** [comment]
> I attempted to implement this, however my attempt appears to have failed. The mod also does not provide any API or any way to open their Death Log screen from another mod. Was planning this for v2.0, but will probaby delay it until later, for when I actually figure out how to get it working.

I do not think it is necessary, in mod "DeathLog" is a configurable button that allows you to quickly get to this menu.

**Comment by Phosphorshy** [comment]
@gliscowo is there any way you could add a way for other mods to add DeathLog support?

**Comment by TheCSDev** [comment]
I plan on implementing a proper API into this mod in the next version I am making, allowing other mods to pull this off, and allowing other mods to implement their own GUI buttons and other features that do stuff. So hopefully, that'll be useful.

**Comment by TheCSDev** [comment]
`v3.0` is planned to be complete some time soon, and will feature an API. Given the mod in question does not have an API of its own (at the time I checked and as far as I know), I cannot reliably implement this feature. That mod however, will be able to use this mod's API to implement that feature. Will close as "Not planned".
